src/assets/scss/introjs.scss

```diff
@@ -0,0 +1,103 @@
+.introjs-tooltip {
+    background-color: white;
+    border-radius: 9px;
+}
+.introjs-helperNumberLayer{
+    display: none;
+}
+.introjs-helperLayer{
+    border-radius: 9px;
+}
+.introjs-donebutton{
+    color: white !important;
+    background-image: none !important;
+    background-color: #508197 !important;
+    text-shadow: none;
+    border: none;
+    border-radius: 3px;
+    float: right !important;
+    font-family: "ProximaNova";
+    padding: .5em .7em .5em !important;
+}
+.introjs-donebutton:hover{
+    color: white !important;
+    background-color: #508197;
+    background-image: none;
+}
+.introjs-donebutton:focus{
+    color: white !important;
+    background-color: #508197;
+    background-image: none;
+}
+.introjs-skipbutton{
+    float:left;
+    border:none;
+    background-image: none;
+    font-family:"ProximaNova"; 
+    padding-left: 0px;
+    background-color: white;
+    color: #508197;
+}
+.introjs-skipbutton:active{
+    background-image: none;
+    background-color:transparent;
+    font-family:"ProximaNova"; 
+    padding-left: 0px;
+    color: #508197;
+    background-color: white;
+}
+.introjs-skipbutton:focus{
```

Comment:
Since this :focus is identical to the :active, you can combine them! There are some great examples of the best way to do this in SCSS in Navbar.vue